tests/ClientWrapperTest.php

```diff
@@ -91,4 +92,47 @@ public function testGetBranchName(): void
             $branchesApi->deleteBranch((int) $branchId);
         }
     }
+
+    public function testClientsAreProperlyConfigured(): void
+    {
+        $backendConfiguration = new BackendConfiguration('123-transfomration', 'small');
+
+        $clientOptionsMock = $this->createMock(ClientOptions::class);
+        $clientOptionsMock->expects(self::exactly(2))
+            ->method('getClientConstructOptions')
+            ->willReturn([
+                'url' => 'dummy',
+                'token' => 'dummy-token',
+            ]);
+
+        $clientOptionsMock->expects(self::exactly(2))
```

Comment:
No jako podle názvu test, dejme tomu, ale reálně je podle mě k ničemu - testuje, že když si dvakrát zavolám mock, co vrací 124, tak to dvakrát vrátí 124.